### PR TITLE
Add ability to apply color scale on columns with numeric values

### PR DESCRIPTION
**Pull request by VkoHov**
Add ability to apply color scale in Runs explore

### CODE REVIEW

**Comment by VkoHov** [comment]
Resolves [#893](https://github.com/aimhubio/aim/issues/893)

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 - Hugging Face adapter refactoring (mihran113)
 - Add run description columns to all run specific tables (VkoHov, mihran113)
 - Change images rendering optimization default value to smoother (VkoHov)
+- Add ability to apply color scaleon columns with numeric values (VkoHov)
```
Comment (Hamik25) [inline CHANGELOG.md:10]: Please fix the typo "scaleon -> scale on"

File: aim/web/ui/src/components/CustomTable/TableColumn.tsx [hunk 2]
```diff
@@ -235,6 +272,25 @@ function Column({
                 )}
                 component={
                   <div className='Table__action__popup__body'>
+                    {!_.isEmpty(colorScaleRange) && onToggleColumnsColorScales && (
+                      <MenuItem
+                        className='Table__action__popup__item'
+                        onClick={() => onToggleColumnsColorScales(col.key)}
+                      >
+                        <span className='Table__action__popup__item_icon'>
+                          {columnsColorScales[col.key] ? (
```
Comment (Hamik25) [inline aim/web/ui/src/components/CustomTable/TableColumn.tsx:281]: You can avoid double-checking here.

**Review by Hamik25 — CHANGES_REQUESTED** [review]

**Review by rubenaprikyan — COMMENTED** [review]
@VkoHov  The first thing comes to mind is, extend this functionality in other explorers, thoughts?
I thought having this in metrics explorer is valuable as well.

**Review by rubenaprikyan — APPROVED** [review]
lgtm

**Pull request merged**